dev_guide/builds.adoc

```diff
@@ -1246,11 +1246,25 @@ $ oc start-build hello-world --from-repo=../hello-world --commit=v2
 
 == Canceling a Build
 Manually cancel a build using the web console, or with the following CLI command:
-
 ----
 $ oc cancel-build <build_name>
 ----
 
+Cancel multiple builds at the same time:
+----
+$ oc cancel-build <build1_name> <build2_name> <build3_name>
+----
+
+Cancel all builds created from the build configuration:
+----
+$ oc cancel-build bc/foo
+----
+
+Specify what builds should be cancelled:
+----
+$ oc cancel-build bc/foo --state=new
```

Comment:
I had some questions about what `--state=new` really meant / would do, or if there were other valid values. There's some more info in https://github.com/openshift/origin/pull/8509 (in Discussion + Files Changed) that might be worth pulling into this dev_guide topic (the cli_reference topic seems fine to keep less verbose).